proxy/src/serverless/conn_pool.rs

```diff
@@ -425,8 +431,15 @@ async fn connect_to_compute(
         application_name: Some(APP_NAME),
         options: console_options.as_deref(),
     };
-
-    let node_info = creds
+    // TODO(anna): extra call to control plane might slow down everything add caches.
```

Comment:
Repeating my earlier comment. This part is crucial. We need serverless driver to be quick (We promise <10ms).

I would say this is a blocker for this PR. We need a cache for this

Reply:
Added simple ttl-lru cache. Adding something more complicated requires non-trivial refactoring (because in rust it's not so easy to create a detached asynchronous task to fetch ips in background).

Probably, at this point it's better to implement a proper solution with redis.